src/components/CryptoSelectForm.js

```diff
@@ -5,15 +5,18 @@
 		---The main reason why there is still this component and the IntlSelectForm is
 		because I couldn't get the crypto.name to render conditionally while being mapped.
 */
-
+/*
+	I added the animation and delay styles which are passed in via props from
+	the CryptoTraq component.
+ */
 import React from 'react';
 import Radium from 'radium';
 import RadiumVars from '../RadiumVariables';
 
 
 const CryptoSelectForm = (props) =>{
 	return (
-		<select style={styles.selectForm__form}>
+		<select style={[styles.selectForm__form, props.delay, props.anim]}>
```

Comment:
You can add any delay and keyframe combination you want to this select element now. I am using it to draw attention to the element to show that the user needs to take an action on this element.